packages/runtime/src/StoryboardFunctionRegistry.ts

```
@@ -8,6 +12,7 @@ import {
 } from "@next-core/cook";
 import { supply } from "@next-core/supply";
 import { collectMemberUsageInFunction } from "@next-core/utils/storyboard";
+import { hasOwnProperty } from "@next-core/utils/general";
```

Biome (1.9.4)

[error] Do not shadow the global "hasOwnProperty" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)